models/src_shopify.yml

```diff
@@ -7,9 +7,9 @@ sources:
     tables:
       - name: order
         description: Each record represents an order in Shopify.
-        identifier: "{%- if target.type == 'snowflake' -%} ORDER {%- else -%} order {%- endif -%}"
-        quoting:
-          identifier: true
+        identifier: >
+          {% if target.type == 'snowflake' %}{{ var('shopify_order_identifier', '"ORDER"') }}
+          {% else %}{{ var('shopify_order_identifier', 'order') }}{% endif %}
```

Comment:
is quoting necessary for the other warehouses though?

Reply:
like should it be 
```suggestion
        identifier: "{%- if target.type == 'snowflake' -%} {{ var('shopify_order_identifier', 'ORDER' ) }} {%- else -%} {{ var('shopify_order_identifier', 'order') }} {%- endif -%}"
        quoting:
          identifier: true
```

Reply:
Made a small change since the identifier was not working with snowflake when quoting was turned on. It's passing now!